pkgs/games/angband/default.nix

```diff
@@ -1,37 +1,67 @@
 { lib, stdenv, fetchFromGitHub, autoreconfHook, ncurses5
 , enableSdl2 ? false, SDL2, SDL2_image, SDL2_sound, SDL2_mixer, SDL2_ttf
+, darwin
 }:
+stdenv.mkDerivation (finalAttrs: {
+  inherit enableSdl2;
 
-stdenv.mkDerivation rec {
   pname = "angband";
   version = "4.2.5";
 
   src = fetchFromGitHub {
     owner = "angband";
     repo = "angband";
-    rev = version;
+    rev = finalAttrs.version;
     sha256 = "sha256-XH2FUTJJaH5TqV2UD1CKKAXE4CRAb6zfg1UQ79a15k0=";
   };
 
-
-  nativeBuildInputs = [ autoreconfHook ];
-  buildInputs = [ ncurses5 ]
-  ++ lib.optionals enableSdl2 [
-    SDL2
-    SDL2_image
-    SDL2_sound
-    SDL2_mixer
-    SDL2_ttf
-  ];
-
-  configureFlags = lib.optional enableSdl2 "--enable-sdl2";
-
-  installFlags = [ "bindir=$(out)/bin" ];
-
   meta = with lib; {
```

Comment:
Please keep this at the bottom of the file. That will make the diff more readable.